modules/astraoneBidAdapter.js

```diff
@@ -0,0 +1,140 @@
+import * as utils from '../src/utils'
+import { registerBidder } from '../src/adapters/bidderFactory'
+import { BANNER } from '../src/mediaTypes'
+
+const BIDDER_CODE = 'astraone';
+const SSP_ENDPOINT = 'https://ssp.astraone.io/auction/prebid';
+const TTL = 60;
+
+function buildBidRequests(validBidRequests) {
+  return utils._map(validBidRequests, function(validBidRequest) {
+    const params = validBidRequest.params;
+    const bidRequest = {
+      bidId: validBidRequest.bidId,
+      transactionId: validBidRequest.transactionId,
+      sizes: validBidRequest.sizes,
+      placement: params.placement,
+      placeId: params.placeId,
+      imageUrl: params.imageUrl
+    };
+
+    return bidRequest;
+  })
+}
+
+function buildBid(bidDada) {
+  const bid = {
+    requestId: bidDada.bidId,
```

Comment:
Is `bidDada` typo? If yes, can you change it to `bidData`.

Reply:
Hi, thanks for your comment. We fixed that typo.